src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -14,6 +14,7 @@
             [liberator.core :refer [defresource]]
             ;[liberator.dev :as liberator-dev]
```

Comment:
@dankreek it seems like we've been leaving this in along with https://github.com/puppetlabs/puppet-server/pull/603/files#diff-bb1d8f2d842888913c49c1dddc70a5eaR263 because it's really helpful for debugging? It's at least been here a while, I think.